api/src/main/resources/META-INF/native-image/io.jsonwebtoken/jjwt-api/resource-config.json

```diff
@@ -0,0 +1,9 @@
+{
+  "resources": {
+    "includes": [
+      {
+        "pattern": "META-INF/services/io.jsonwebtoken.CompressionCodec"
```

Comment:
There are a handful of services, though these should _possibly_ live in their corresponding module, and not in `jjwt-api`, thoughts?

```txt
./impl/src/main/resources/META-INF/services/io.jsonwebtoken.CompressionCodec
./extensions/gson/src/main/resources/META-INF/services/io.jsonwebtoken.io.Serializer
./extensions/gson/src/main/resources/META-INF/services/io.jsonwebtoken.io.Deserializer
./extensions/orgjson/src/main/resources/META-INF/services/io.jsonwebtoken.io.Serializer
./extensions/orgjson/src/main/resources/META-INF/services/io.jsonwebtoken.io.Deserializer
./extensions/jackson/src/main/resources/META-INF/services/io.jsonwebtoken.io.Serializer
./extensions/jackson/src/main/resources/META-INF/services/io.jsonwebtoken.io.Deserializer
```

Reply:
I put it all in -api for maintaner to maintain them easier, no need to worry about what modification shall look for which json
but yes, logically it should be split into each module

Reply:
I would like to try to write a maven-plugin to auto-generate it, so the two problems shall both be solved.